functests/performance/rt-kernel.go

```diff
@@ -59,4 +60,19 @@ var _ = Describe("[performance]RT Kernel", func() {
 
 	})
 
+	It("[test_id:28526][crit:high][vendor:cnf-qe@redhat.com][level:acceptance] Non worker-cnf node should not have RT kernel installed", func() {
+
+		By("Skipping test if cluster does not have another available worker node")
+		nonRTWorkerNode, err := nodes.GetNonRTWorkers()
```

Comment:
nit: please use plural for the var as well, avoids confusion (sorry, missed it in earlier review)